metricbeat/module/kafka/broker.go

```diff
@@ -270,6 +280,15 @@ func (b *Broker) FetchGroupOffsets(group string, partitions map[string][]int32)
 	return b.broker.FetchOffset(requ)
 }
 
+// GetPartitionOffsetFromTheLeader fetches the OffsetNewest from the leader.
+func (b *Broker) GetPartitionOffsetFromTheLeader(topic string, partitionID int32) (int64, error) {
```

Comment:
Nit. Use `Fetch` for consistency with other methods here.
```suggestion
func (b *Broker) FetchPartitionOffsetFromTheLeader(topic string, partitionID int32) (int64, error) {
```